### DISCUSSION TITLE
SatPI on GigaBlue Quad4k with DVB-S2XFBC - not working on unicable mode

### DISCUSSION THREAD

**Discussion by ddarek** [Barracuda09/SATPI#198] [Q&A]
Hi,
I have a tuner GigaBlue Quad4K with DVB-S2.
I have an openatv image. I have disabled enigma2 run, installej satpi (from opkg) and tvheadend.
My tuner is connected to lnb with UnicableII.
On legacy connection to this lnb (without unicable) everything works but when I connect it using Unicable configuration then there is no data received.
On Enigma it's working correctly, but on SatPI it's not working.
On frontend monitor info I see:
Status=1
Signal stregth =144
Snr=6
Ber=0
Unc=0
Pid=0,16,17,18
CCerrors=0.

I tried on different unicable channels but without luck. Also tried only with one frontend enabled.
When I change it to legacy mode and with diseqc it's working.
Unicable mode works when I switch to enigma.

Could someone tell me if I need something more to do to have it working. I configured even with exact same configs like in instruction that is available on wiki page.

I attach logs from SatPi when I turned on scanning on hotbird:
[sat_pi_log.txt](https://github.com/Barracuda09/SATPI/files/12497733/sat_pi_log.txt)

Thanks in advance
Darek

**Comment by Barracuda09** [comment]
Hi Darek,

Could you give some more details of the unicable LNB and what you have setup in SatPI.?

**Reply by ddarek** [reply]
Hi,
I attach the zipped xml file with SatPI configuration.
I'm using Opticum Unicable II SCR 24UB 3legacy.
Channels:  1210/1420/1680/2040/978/1028/1072/1116/1160/1271/1315/1359/1484/1528/1572/1616/2084/2128/1816/1860/1904/1948/1992/1728
SCR 8 / EN50494+EN50607 + 16 / EN50607

[SCR_24UB_0](https://github.com/Barracuda09/SATPI/assets/9989513/726cd298-ab30-491c-a27d-c882ca5905ce)
[SatPI.xml.zip](https://github.com/Barracuda09/SATPI/files/12497923/SatPI.xml.zip)

Best regards
Darek

**Reply by Barracuda09** [reply]
I see in the xml this for frontend 1:
```
<chFreq>
<inputtype>number</inputtype>
<value>1072</value>
<minvalue>0</minvalue>
<maxvalue>2150</maxvalue>
</chFreq>
<chSlot>
<inputtype>number</inputtype>
<value>0</value>
<minvalue>0</minvalue>
<maxvalue>31</maxvalue>
</chSlot>
```
Here chSlot is 0 and chFreq is 1072 -> this should probably be chFreq 1210?

In the log I see this:
```
  SETUP rtsp://10.0.0.6/stream=100?src=1&fe=1&freq=11642&sr=27500&msys=dvbs2&mtype=8psk&pol=h&fec=34&ro=0.20&plts=off&pids=0 RTSP/1.0
```
src=1 does this mean there is a diseqc switch?

**Reply by ddarek** [reply]
Hi,
I was trying with different channels. 1210 is the first one. With 1210 I had the same value.
I don't have diseqc whe I use unicable. Tuner is conected directly to LNB in unicable mode. I have diseqc only in legacy connection.
I saw that there is this param src=1 and there was in log file that diseqc is chosen. 
Tvheadend is generating this even when I have only one network configured.
Could this be the cause?

**Reply by ddarek** [reply]
On a different tuner with unicable I have src=1 and it works

**Comment by Barracuda09** [comment]
You should use:
```
chFreq: 1210 and chSlot: 0
```
The count starts with chSlot 0 for first SCR channel

----

You could try this in a browser (or wget) and see the log again:
```
http://10.0.0.6:8875/stream=100?&fe=1&freq=11642&sr=27500&msys=dvbs2&mtype=8psk&pol=h&fec=34&ro=0.20&plts=off&pids=0,16,17,18
```

At least as this is a valid channel on hotbird

**Reply by ddarek** [reply]
Ah, 
I was using chSlots from 1 not from 0.
This change and the right freq and it works :)

Now it seems to be ok.

Thanks for your help
Darek

**Reply by ddarek** [reply]
It started to work, but when I enabled other frontends then it stoped.
After restart again it's not working - only for a while it was working.

I attached my current config + clean log from the start to moment when I try to receive something using the url you gave.

[satpi.zip](https://github.com/Barracuda09/SATPI/files/12498858/satpi.zip)

**Reply by Barracuda09** [reply]
Could you try to add the **src=1** like so:
```
  http://10.0.0.6:8875/stream=100?src=1&fe=1&freq=11642&sr=27500&msys=dvbs2&mtype=8psk&pol=h&fec=34&ro=0.20&plts=off&pids=0,1,16,17,18
```

Maybe try also without **ro=** and **plts=** 

Thanks for trying and your feedback

**Reply by ddarek** [reply]
Hi Mark!
I have solved the problem.
I needed to select checkbox "Send DiSEqC Via Linked Root Tuner" on all tuners.
I'm not using DiSEqC on unicable configuration so I was not selecting this option. But it seems that it's required.

I attach my SatPI.xml file with right configuration :)

Thanks for your support
Darek

[SatPI.xml.zip](https://github.com/Barracuda09/SATPI/files/12504544/SatPI.xml.zip)

**Comment by Barracuda09** [comment]
Yes for FBC tuners the DiSEqC commands are sometimes not redirected to the root tuner. That is why I added this option **"Send DiSEqC Via Linked Root Tuner"**

Unicable uses DiSEqC commands to setup the LNBs UB
